src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -803,13 +803,24 @@ void Engine::ApplyPerformanceHints(std::map<std::string, std::string> &config, c
     hints_props.insert({tput_name, tput_hints.second});
     ngraphFunc->set_rt_info(hints_props, "intel_cpu_hints_config");
 
+    auto resetHybridParam = [&]() {
+        config[CONFIG_KEY_INTERNAL(BIG_CORE_STREAMS)] = std::to_string(0);
+        config[CONFIG_KEY_INTERNAL(SMALL_CORE_STREAMS)] = std::to_string(0);
+        config[CONFIG_KEY_INTERNAL(THREADS_PER_STREAM_BIG)] = std::to_string(0);
+        config[CONFIG_KEY_INTERNAL(THREADS_PER_STREAM_SMALL)] = std::to_string(0);
+        config[CONFIG_KEY_INTERNAL(SMALL_CORE_OFFSET)] = std::to_string(0);
+    };
+
     const auto perf_hint_name = getPerfHintName();
     if (perf_hint_name == CONFIG_VALUE(LATENCY)) {
         config[CONFIG_KEY(CPU_THROUGHPUT_STREAMS)] = latency_hints.first;
         config[ov::num_streams.name()] = latency_hints.second;
+        resetHybridParam();
```

Comment:
Don't think this is appropriate solution. I would say changing content of CPU plugin config inside getter "GetHybridNumStreams" is not a good programming way. 
From my understanding these hybrid related keys (like BIG_CORE_STREAMS) should be part only of streamExecutorConfig and should be initialized inside MakeDefaultMultiThreaded